integration-tests/src/test/java/oracle/weblogic/domain/ServerStatus.java

```diff
@@ -27,9 +28,19 @@
   @ApiModelProperty("Name of node that is hosting the Pod containing this WebLogic Server.")
   private String nodeName;
 
+  @ApiModelProperty("Phase of server pod. Possible values are: Pending, Succeeded, Failed, Running, or Unknown.")
+  @Expose
+  private String podPhase;
+
```

Comment:
server -> WebLogic Server 

ditto for podReady

(follows the pattern of the the other status field descriptions)